config/initializers/doorkeeper.rb

```diff
@@ -11,15 +11,6 @@ module Doorkeeper
   autoload :Application, 'doorkeeper/orm/active_record/application'
   autoload :BaseRecord, 'doorkeeper/orm/active_record/base_record'
   autoload :RedirectUriValidator, 'doorkeeper/orm/active_record/redirect_uri_validator'
-  module Orm
-    module ActiveRecord
-      class << self
-        undef initialize_models!
-        def initialize_models!
-        end
-      end
-    end
-  end
```

Comment:
Just curious why this needed to be deleted.

Reply:
This function no longer exists in the Doorkeeper gem, so we don't need to replace it with an empty method.